app.css

```diff
@@ -1,3 +1,88 @@
-body {
-  background: purple;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+
+.blog__post {
+  width: 44%;
```

Comment:
Change this to a calc width. 44% will never be able to properly grid the items on this page when the margins are in `rem` values (which should still remain `rem` values).